.github/pull_request_template.md

```diff
@@ -6,12 +6,13 @@ This PR fixes/adds/improves/...
 
 ### Steps
 
-<!-- please tick steps this PR performs (if something is not necessary, please tick anyway to indicate you considered it) -->
+<!--please tick steps this PR performs (if something is not necessary, please remove it) -->
 
 - [ ] added implementation
 - [ ] added / updated tests
 - [ ] added / updated documentation
 - [ ] extended changelog
+- [ ] I read the `CONTRIBUTING.md` and signed-off my commits to accept the DCO.
```

Comment:
```suggestion
- [ ] I read the [contribution documentation](https://github.com/hedgedoc/hedgedoc/blob/master/CONTRIBUTING.md) and signed-off my commits to accept the DCO.
```
Maybe link directly to the docs?

Reply:
Should we link to the DCO then too?

Reply:
We could, but the contribution docs one is supposed to read already do :D

Reply:
fixed